docs/getting-started-guides/docker-multinode.md

```diff
@@ -0,0 +1,261 @@
+### Running Multi-Node Kubernetes Using Docker
+
+_Note_: These instructions are somewhat significantly more advanced than the [single node](docker.md) instructions.  If you are
+interested in just starting to explore Kubernetes, we recommend that you start there.
+
+
```

Comment:
I think it might be helpful to add a note about the setup and pattern here: The following guide will make use of two docker daemons, a pattern which will allow us to manage and deploy "system" services such as flannel and etcd in containers...